app/helpers/crypt.js

```diff
@@ -0,0 +1,15 @@
+const bcrypt = require('bcrypt');
```

Comment:
Keep in mind the format of the requires. You can see it here: [Wolox NodeJS Standards and best practices](https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#82--requires)